.gitignore

```diff
@@ -19,3 +19,6 @@ doc/api/
 *.js_
 *.js.deps
 *.js.map
+
+# IDEA
+.idea/
```

Comment:
This project has nothing to do with IDEA in particular. Developer-specific stuff belongs in the personal global `.gitignore`.

Reply:
I add things like this to local `.gitignore`)

Reply:
So what is the reason for adding it here?